components/brave_ads/common/pref_names.cc

```diff
@@ -16,9 +16,9 @@ const char kAdsWereDisabled[] = "brave.brave_ads.were_disabled";
 // "No Wallet".
 const char kHasAdsP3AState[] = "brave.brave_ads.has_p3a_state";
 
-// Stores ad view confirmation count
-const char kAdViewConfirmationCountPrefName[] =
-    "brave.weekly_storage.p2a_ad_view_confirmation_count";
+// Prefix for pref name to store the count of ad impressions per segment
```

Comment:
Should this comment be generic as this prefix will be used for any ads p2a event

Reply:
changed